content/handbook/support/license-and-renewals/workflows/working_internal_requests.md

```
@@ -0,0 +1,32 @@
+category: Working L&R requests
+description: ライセンス & 更新に関する内部リクエストの対応方法を説明します。
+upstream_path: /handbook/support/license-and-renewals/workflows/working_internal_requests/
+upstream_sha: 9da08370100b76603b2eb82901592f9d21c9766c
+translated_at: "2026-05-08T11:18:50Z"
+translator: claude
+stale: false
+---
+
+---
+
+### 内部リクエストの対応
```

markdownlint-cli2 (0.22.1)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)